lib/msal-common/src/cache/entities/AccountEntity.ts

```diff
@@ -236,12 +247,18 @@ export class AccountEntity {
      * @param serverClientInfo
      * @param authType
      */
-    static generateHomeAccountId(serverClientInfo: string, authType: AuthorityType, logger: Logger, cryptoObj: ICrypto, idToken?: AuthToken): string {
+    static generateHomeAccountId(
+        serverClientInfo: string,
+        authType: AuthorityType,
+        logger: Logger,
+        cryptoObj: ICrypto,
+        idToken?: AuthToken
+    ): string {
 
         const accountId = idToken?.claims?.sub ? idToken.claims.sub : Constants.EMPTY_STRING;
 
         // since ADFS does not have tid and does not set client_info
-        if (authType === AuthorityType.Adfs) {
+        if (authType === AuthorityType.Adfs || authType === AuthorityType.Dsts) {
```

Comment:
Does dSTS also not have the concept of tenants?

Reply:
dSTS does have tenants, but afaik they do not support guest accounts and client_info probably isn't emitted. @neha-bhargava, do you know?

Reply:
I can't remember where I read it but I'm fairly certain I did read client_info wasn't emitted (and it's not returned in the Fiddler trace either).